src/libs/TransactionUtils.ts

```diff
@@ -337,8 +337,11 @@ function isExpensifyCardTransaction(transaction: Transaction): boolean {
 /**
  * Determine whether a transaction is made with a card.
  */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 function isCardTransaction(transaction: Transaction): boolean {
-    return (transaction?.cardID ?? 0) > 0;
+    // We are returning cash cardID via the API. We can undo this after it is fixed in Auth.
+    // return (transaction?.cardID ?? 0) > 0;
+    return false;
```

Comment:
I think this might introduce another issue. We use this logic to decide if transactions are [editable](https://github.com/Expensify/App/blob/61ed9009dfcb1882ac540550191777e143575bd8/src/components/ReportActionItem/MoneyRequestView.js#L113), so always returning false means users will see card transactions as editable in the UI. Can we check if the card is a CASH card instead?

Reply:
>Can we check if the card is a CASH card instead?

Yes, that would be the "real" fix. Or are you saying we are returning that somewhere already?

Reply:
We can maybe fix it in the PHP layer and CP that... ?

Reply:
Hmm I think not - it is still missing some things to determine if it is the cash card. Like - `bank`, `managedCard` etc

https://github.com/Expensify/Web-Expensify/blob/51a427f541a8b38843bed21d65ac97c4b6c7f93f/lib/Transaction.php#L1519-L1535

Reply:
WRT to editing card transactions. Do we prevent that in the API? Them being editable might be weird, but I'd take that bug over this one.

Reply:
It's also... what we have on production already? 😄

Reply:
I don't think this is a problem with what we're sending- let's just filter out cash cards in this function?

Reply:
@grgia I like the solution you came up with to fix the Deploy Blocker. 

In web-expensify it looks like we download all cards and then check to see if the card is cash by looking at the `cardName`, `bank` etc. Having something like `isCashCard` returned from Auth and attached to the transaction might be more direct.